index.bs

```diff
@@ -8546,6 +8564,23 @@ entails.
 
 <h4 id="LegacyWindowAlias" extended-attribute lt="LegacyWindowAlias">[LegacyWindowAlias]</h4>
 
+<div class="advisement">
+
+    [{{LegacyWindowAlias}}] [=extended attributes=] are an undesirable feature.
+    They exist only so that legacy Web platform features can be specified.
+    They should not be used in specifications
+    unless required to specify the behavior of legacy APIs.
+
+    <small class="non-normative">
+        The [{{LegacyWindowAlias}}] [=extended attribute=] appears on the following [=interfaces=]:
```

Comment:
I am not really in favor of this because maintaining a reverse-registry kind of defeats the point of Web IDL (in creating a toolset anyone can use).

Listing maybe one example is fine. But saying "here are all the places it appears" exhaustively is not good and will either be out of date or interpreted as a requirement to add your use to here.